app/controllers/index.js

```diff
@@ -60,4 +60,23 @@ export default Ember.Controller.extend(Analytics, {
 
         this.set('currentDate', new Date());
     },
+
+    actions: {
+        contactLink(href, category, action, label) {
+            const metrics = Ember.get(this, 'metrics');
+
+            // TODO submit PR to ember-metrics for a trackSocial function for Google Analytics. For now, we'll use trackEvent.
+            metrics.trackEvent({
+                category,
+                action,
+                label
+            });
+
+            if (label.includes('email'))
+              return;
+
+            window.open(href, '', 'menubar=no,toolbar=no,resizable=yes,scrollbars=yes,width=600,height=400');
```

Comment:
Hmm.... In general, MDN recommends against using this method:
https://developer.mozilla.org/en-US/docs/Web/API/Window/open#Avoid_resorting_to_window.open()

It prefers that developers use links directly as intended. What is the purpose of opening a new window for the emails?

It appears that href is filled in here; there's a chance of user-entered text (node.title) becoming part of the link url. That's not great, though it appears the characters are escaped in this case:
https://github.com/abought/ember-preprints/blob/381fbb8e6017a0553d66a77d217825721a4c46c8/app/controllers/content.js#L96-L96

We should avoid using `window.open` and prefer an anchor link directly if possible, for reasons cited by MDN above. If we must use this method, the value of href should be treated carefully, and features such as `noopener` should be very explicitly used.